project/src/pages/MainPage/index.tsx

```diff
@@ -0,0 +1,80 @@
+import { FilmCardMain } from '../../components/FilmCardMain';
+import { FilmList } from '../../components/FilmList';
+import { Footer } from '../../components/Footer';
+
+type Film = {
```

Comment:
Is it same type as Film in `project/src/types/film.ts`? Should we delete it?

Reply:
done